templates/partners/iot-device.html

```diff
@@ -18,13 +19,22 @@ <h1>IoT Device Programme</h1>
       <p><a href="/partners/become-a-partner" class="p-button--positive">Become a partner</a></p>
     </div>
     <div class="col-6 u-align--center u-hide--small u-hide--medium">
-      <img src="https://assets.ubuntu.com/v1/a491f87c-Ubuntu+Certified+Hardware.svg" style="max-width:364px;" alt="">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/a491f87c-Ubuntu+Certified+Hardware.svg",
+        alt="",
+        width="394",
+        height="350",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+      }}
     </div>
   </div>
 </section>
-<section class="p-strip is-deep">
+
+<section class="p-strip has-accent-border--top-left is-deep">
   <div class="u-fixed-width u-align--center">
-    <p class="p-logo-section__title" style="max-width: none;">A selection of our IoT partners </p>
```

Comment:
This is no longer centered:
![Screenshot 2022-04-28 at 12 40 28](https://user-images.githubusercontent.com/17607612/165735241-8f596568-e07a-4106-b905-9cadbec374ef.png)

Reply:
Thank you and good catch!